src/common/identity_parser.hpp

```diff
@@ -26,6 +26,8 @@ namespace kvp = mender::common::key_value_parser;
 
 kvp::ExpectedKeyValuesMap GetIdentityData(const string &identity_data_generator);
 
+string DumpIdentityData(kvp::KeyValuesMap identity_data);
```

Comment:
`const kvp::KeyValuesMap &identity_data`

Reply:
One issue if I take the const reference is that I cannot do operator '[] `

```
/home/lluis/northern.work/mender/src/common/identity_parser/identity_parser.cpp:49:38: error: passing ‘const KeyValuesMap’ {aka ‘const std::unordered_map<std::__cxx11::basic_string<char>, std::vector<std::__cxx11::basic_string<char> > >’} as ‘this’ argument discards qualifiers [-fpermissive]
   49 |                 if (identity_data[key].size() == 1) {
      |                                      ^
```

So I'll use non-const reference :+1:

Reply:
Why not use `.at` in any case?

Reply:
Thank you :see_no_evil: